src/wallet/wallet2.cpp

```diff
@@ -14289,12 +14287,14 @@ size_t wallet2::import_multisig(std::vector<cryptonote::blobdata> blobs)
     }
     seen.insert(signer);
 
-    std::string body(data, headerlen);
-    std::istringstream iss(body);
+    std::string_view body{data};
```

Comment:
The old code was using std::string's substring constructor (`basic_string(const basic_string& other, size_type pos, size_type count = std::basic_string::npos)`), so ends up with `data` starting at position `headerlen`.  The new code does the same thing by making a full-size string_view and then, in the next line, chopping off `headerlen` from the view.

Reply:
(This string constructor seems really unintuitive since `string(foo, 6)` does completely different things when `foo` is a `const char*` rather than a `std::string`).

Reply:
ah yeah. Got confused thinking it was starting at data and ending at headerlen.